### PR TITLE
Blog: seen by in Postgres

### PR DESCRIPTION
**Pull request by kiwicopple**
https://zone-www-dot-com-git-blog-vadosseenby-supabase.vercel.app/blog/2022/07/12/seen-by-in-postgresql

### CODE REVIEW

File: apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx [hunk 1]
```diff
@@ -0,0 +1,736 @@
+---
+title: Implementing "seen by" functionality with Postgres
+description: Different approaches for tracking visitor counts with PostgreSQL.
+author: paul_copplestone
```
Comment (kiwicopple) [inline apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx:4]: todo: change this to Vados (need to include his details in the authors page)

File: apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+image: 2022-june/thumb.png
+thumb: 2022-june/thumb.png
```
Comment (kiwicopple) [inline apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx:6]: Need new images here

File: apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx [hunk 3]
```diff
@@ -0,0 +1,736 @@
...
+[wiki-bloom-filter]: https://en.wikipedia.org/wiki/Bloom_filter
+[wiki-hll]: https://en.wikipedia.org/wiki/HyperLogLog
+[pg-cron]: https://github.com/citusdata/pg_cron
+[pg-fdw]: https://www.postgresql.org/docs/current/postgres-fdw.html
+[pg-view]: https://www.postgresql.org/docs/14/sql-createview.html
+[pg-tablespaces]: https://www.postgresql.org/docs/current/manage-ag-tablespaces.html
+[repo-scripts/generate/users.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/generate/users.js
+[repo-scripts/generate/posts.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/generate/posts.js
+[repo-scripts/bench.js]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/bench.js
+[repo-scripts/setup-request.cjs]: https://gitlab.com/mrman/supabase-seen-by/-/blob/main/scripts/setup-request.cjs
```
Comment (kiwicopple) [inline apps/www/_blog/2022-07-12-seen-by-in-postgresql.mdx:736]: Should we move all of these into a repo here? https://github.com/supabase-community/

**Comment by steve-chavez** [comment]
Is there a pre-rendered view of this post?

Also, the javascript part makes it a bit complex(lots of references).

Is that part necessary? Perhaps it could it be done with pure SQL + pgbench.

**Comment by MildTomato** [comment]
weird, i can't push to this branch 🤨

**Review by RamiroND — APPROVED** [review]

**Pull request merged**